nvOC_miner_update.sh

```diff
@@ -49,21 +49,21 @@ if [ ! $(cat ${NVOC_MINERS}/ewbf/latest/version | grep v0.3) ]
 then
   echo "Extracting EWBF Equihash miner"
   mkdir -p ${NVOC_MINERS}/ewbf/{latest,v0_3,3_4,3_3}
-  stop-if-needed "[m]iner"
   cat ${NVOC_MINERS}/ewbf/0.3.4b.tar.xz	.tar.xz | tar -xJC ${NVOC_MINERS}/ewbf/3_4/ --strip 1
   cat ${NVOC_MINERS}/ewbf/0.3.3b.tar.xz	.tar.xz | tar -xJC ${NVOC_MINERS}/ewbf/3_3/ --strip 1
   cat ${NVOC_MINERS}/ewbf/v0.3.tar.xz	.tar.xz | tar -xJC ${NVOC_MINERS}/ewbf/v0_3/ --strip 1
   chmod a+x ${NVOC_MINERS}/ewbf/v0_3/miner
   chmod a+x ${NVOC_MINERS}/ewbf/3_4/miner
   chmod a+x ${NVOC_MINERS}/ewbf/3_3/miner
-  restart-if-needed
+  stop-if-needed "[m]iner"
```

Comment:
We have a problem here, "[m]iner" will match pretty always, use "[e]wbf" instead, should match the path instead of the process name

Reply:
Yup, that caught my eye too, is it possible to use it like "[e]wbf/*/miner" or just "[e]wbf"? 
Or we can change its name to "ewbf_miner"

Reply:
"[e]wbf", the folder name,  will catch any of them because they're all in the same folder. Since you're extracting all ewbf versions in this same code block, it makes sense to catch them all.

… he also took the worst choice in naming the miner executable...